spk/duplicity/Makefile

```diff
@@ -4,7 +4,8 @@ SPK_REV = 1
 SPK_ICON = src/duplicity.png
 
 BUILD_DEPENDS = cross/python cross/setuptools cross/pip cross/wheel
-BUILD_DEPENDS += cross/boto cross/netifaces cross/wrapt cross/$(SPK_NAME)
+BUILD_DEPENDS += cross/boto cross/msgpack-python cross/netifaces cross/rackspace-novaclient
```

Comment:
Should I replace `include ../../mk/spksrc.python-module.mk` by `include ../../mk/spksrc.python-wheel.mk` in msgpack-python's Makefile? Will this not affect the python package?